time_tracker_api/users/users_namespace.py

```diff
@@ -57,7 +66,11 @@ class Users(Resource):
     @ns.marshal_list_with(user_response_fields)
     def get(self):
         """List all users"""
-        return AzureConnection().users()
+        ftm = FeatureToggleManager('bk-user-role-field')
```

Comment:
I would suggest to change `ftm` to the name of the FF name plus the word Toggle.

Reply:
Agree.

Reply:
It seemed a good idea first, but actually the name of the variable became long.
```
bk_user_role_field_toggle = FeatureToggleManager('bk-user-role-field')
if bk_user_role_field_toggle.is_toggle_enabled_for_user():
    return AzureConnection().users_v2()
return AzureConnection().users()
```
Does that seem right to you? I think we can come up with a shorter name.

Reply:
changed to `user_role_field_toggle`